### PR TITLE
8247973: Javadoc incorrect for IdentityArrayList, IdentityLinkedList

### PR DESCRIPTION
**Pull request by anupamdev20**
The documentation for following methods used object equality mentioning 'equals()'. I have replaced the equals() with == operator where necessary with some formattings.

sun.awt.util.IdentityLinkedList#contains
sun.awt.util.IdentityArrayList#contains
sun.awt.util.IdentityArrayList#indexOf
sun.awt.util.IdentityArrayList#lastIndexOf
sun.awt.util.IdentityArrayList#remove(java.lang.Object) 

Kindly review the change.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [x] Change must be properly reviewed

### Issue
 * [JDK-8247973](https://bugs.openjdk.java.net/browse/JDK-8247973): Javadoc incorrect for IdentityArrayList, IdentityLinkedList

### Reviewers
 * [Alexey Ivanov](https://openjdk.java.net/census#aivanov) (@aivanov-jdk - **Reviewer**)
 * [Phil Race](https://openjdk.java.net/census#prr) (@prrace - **Reviewer**) ⚠️ Review applies to 46a065c455c6ab8f980edbb31188e61983c56013

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.java.net/jdk pull/6710/head:pull/6710` \
`$ git checkout pull/6710`

Update a local copy of the PR: \
`$ git checkout pull/6710` \
`$ git pull https://git.openjdk.java.net/jdk pull/6710/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 6710`

View PR using the GUI difftool: \
`$ git pr show -t 6710`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.java.net/jdk/pull/6710.diff">https://git.openjdk.java.net/jdk/pull/6710.diff</a>

</details>

### CODE REVIEW

**Comment by aivanov-jdk** [comment]
Was the previous PR #6694 not good enough?  
The previous PR has comments to the changes thus you should've referenced it when you created the new one.

**Comment by anupamdev20** [comment]
@aivanov-jdk The PR #6694 included HttpURLConnection too. I therefore closed that PR (https://github.com/openjdk/jdk/pull/6694#pullrequestreview-823124988) and initiated new requested with correct files and changes. As I am slightly new to git I did not know how to update the PR, so I filed a new one.

**Comment by aivanov-jdk** [comment]
> @aivanov-jdk The PR #6694 included HttpURLConnection too. I therefore closed that PR ([#6694 (review)](https://github.com/openjdk/jdk/pull/6694#pullrequestreview-823124988)) and initiated new requested with correct files and changes. As I am slightly new to git I did not know how to update the PR, so I filed a new one.

You could've just reverted the changes to `HttpURLConnection` in your branch and pushed. Effectively, it would've removed the change from the PR.

File: src/java.desktop/share/classes/sun/awt/util/IdentityLinkedList.java [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2007, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2007, 2021 Oracle and/or its affiliates. All rights reserved.
```
Comment (aivanov-jdk) [inline src/java.desktop/share/classes/sun/awt/util/IdentityLinkedList.java:2]:
```suggestion
 * Copyright (c) 2007, 2021, Oracle and/or its affiliates. All rights reserved.
```
There should be another comma.

**Review by aivanov-jdk — CHANGES_REQUESTED** [review]

File: src/java.desktop/share/classes/sun/awt/util/IdentityLinkedList.java (hunk 1 above)
Comment (aivanov-jdk) [reply src/java.desktop/share/classes/sun/awt/util/IdentityLinkedList.java:2]: Other than that, the changes look good.

**Review by prrace — APPROVED** [review]

**Review by aivanov-jdk — APPROVED** [review]

**Pull request closed without merging**